sdk/cognitivelanguage/Azure.AI.Language.Conversations/CHANGELOG.md

```diff
@@ -1,16 +1,23 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.0.0-beta.4 (Unreleased)
```

Comment:
Is it Beta and not GA?

Reply:
I moved the beta out of main to release/clu-2022-05-15-preview. main is now GA; or, rather, pre-GA - I'll do a separate PR to make it final after I finish up another PR coming in for archboard feedback.